src/features/addCard/addCard.js

```diff
@@ -19,11 +19,10 @@ const useStyles = makeStyles({
 
 export function AddCard(){
     const dispatch = useDispatch()
-    const cantClick = useSelector((state) => state.addingField)
     const classes = useStyles()
 
     return(
-        <IconButton size="medium" color="black" onClick={!cantClick && dispatch(toggleAddWind())} className={ classes.addPos }>
+        <IconButton size="medium" color="black" onClick={() => {dispatch(openAddDialogue())}} className={ classes.addPos }>
```

Comment:
You'll want to avoid calling arrow functions inline like this.  It's not very performant every time the component is rendered a new reference to the arrow function gets generated.  it's also kinda hard to read and navigate about.  Instead something like this would be better:  

```
export function AddCard(){
    const dispatch = useDispatch()
    const classes = useStyles()
    
    const handleOpenDialogue = () => dispatch(openAddDialogue()); 

    return(
        <IconButton size="medium" color="black" onClick={handleOpenDialogue} className={ classes.addPos }>
            <AddIcon fontSize="large" />
        </IconButton>
    )

}
```